package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "bugsnag-cli",
+  "version": "v1.1.1",
```

I think typically this would use `1.1.1` rather than `v1.1.1` - and should we start with a 0 version until we've launched the tool?

```suggestion
  "version": "0.0.1",
```